site/docs/numbers/about.mdx

```diff
@@ -10,6 +10,14 @@ hide_title: true
 image: ../../static/img/bandwidth-logo.png
 ---
 
+export const Svg = require('../../static/img/icon-numbers.svg').default;
+
+<Svg className='about-image' /> <br/>
+
 The Bandwidth Dashboard API is where you will purchase, port, provision, and manage all of the telephone numbers in your inventory that you use for calls and messages on the Bandwidth network.
 
-We recommend familiarizing yourself with the [Account Structure](./account/structure) and [Account Setup](./account/setup) guides to ensure everything you will need for your provisioning API calls before beginning testing/development. 
+We recommend familiarizing yourself with the [Account Structure](./account/structure) and [Account Setup](./account/setup) guides to ensure everything you will need for your provisioning API calls before beginning testing/development.
```

Comment:
> to ensure everything you will need for your provisioning API calls before beginning testing/development.
Maybe I'm being picky. Doesn't read right to me. Suggestion:
> to ensure everything you will need for provisioning API calls is set up before beginning your testing or development.

Reply:
I like that better 👍